include/envoy/router/router.h

```diff
@@ -700,6 +700,13 @@ class PathMatchCriterion {
  */
 class HttpRouteTypedMetadataFactory : public Envoy::Config::TypedMetadataFactory {};
 
+/**
+ * Options for setting the virtual host ratelimits on the route.
+ */
+enum class VhRateLimitOptions { Override, Include, Ignore };
+
+enum class VhRateLimitOptionsCase { kVhRateLimits, kIncludeVhRateLimits };
```

Comment:
Perhaps would make sense to note this is also deprecated and was added to support b/c with a deprecated config option.